pkg/obfuscator/domain_test.go

```diff
@@ -127,7 +125,60 @@ func TestDomainObfuscator_FileName(t *testing.T) {
 }
 
 func TestBadDomainInput(t *testing.T) {
-	_, err := NewDomainObfuscator([]string{"[mustgather.com"})
+	_, err := NewDomainObfuscator([]string{"[mustgather.com"}, schema.ObfuscateReplacementTypeConsistent)
 	assert.Error(t, err)
 	assert.Contains(t, err.Error(), "failed to generate regex")
 }
+
+func TestDomainObfuscationStatic(t *testing.T) {
+	for _, tc := range []struct {
+		name    string
+		input   []string
+		output  []string
+		domains []string
+		report  map[string]string
+	}{
+		{
+			// These are the test cases with the static domain obfuscation.
+			name:    "Domain with extension",
+			domains: []string{"test.com"},
+			input:   []string{"requests.test.com.log"},
+			output:  []string{"requests." + staticDomainReplacement + ".log"},
+			report: map[string]string{
+				"test.com": staticDomainReplacement,
+			},
+		},
+		{
+			name:    "non-matching domain",
+			domains: []string{"test.com"},
+			input:   []string{"report.test"},
+			output:  []string{"report.test"},
+			report:  map[string]string{},
+		},
+		{
+			name:    "Multiple-Matching Domains",
```

Comment:
I think you rather want to test whether in the same input string, you're catching multiple domains? 
so maybe input: "The first domain is report.test.com and the second domain is example-test.com"

otherwise you just test that golang for loops work

Reply:
Addressed